zeth_contracts/contracts/AltBN128MixerBase_test.sol

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2015-2020 Clearmatics Technologies Ltd
+//
+// SPDX-License-Identifier: LGPL-3.0+
+
+pragma solidity ^0.5.0;
+pragma experimental ABIEncoderV2;
+
+import "./AltBN128MixerBase.sol";
+
+
+// Implementation of abstract AltBN128MixerBase contract, to allow testing
+// specific methods.
+contract AltBN128MixerBase_test is AltBN128MixerBase
+{
+    constructor(uint256 mk_depth)
+        public
+        AltBN128MixerBase(mk_depth, address(0), new uint256[](0))
+    {
+    }
+
+    function assemble_public_values_test(uint256 residual_bits)
+        public pure
```

Comment:
I think `pure` should be on the next line

Reply:
moved to next line